awx/ui_next/src/screens/Template/WorkflowJobTemplateVisualizer/Visualizer.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Understandable if you'd rather handle this in a different PR, but thought I'd mention it just in case:

![Screenshot from 2019-11-22 12-04-43](https://user-images.githubusercontent.com/9753817/69445995-7e2a3a80-0d21-11ea-9246-f8f38027f79e.png)

Long names extend past the node borders.

Reply:
Yea I need to add this back in

Reply:
See https://github.com/ansible/awx/pull/5379/commits/3d74dedd8d5f11ec9ef4d88f10d9e063bddcbeee

<img width="1680" alt="Screen Shot 2019-12-02 at 2 23 40 PM" src="https://user-images.githubusercontent.com/9889020/69988290-5f8d2600-150f-11ea-9867-9169ecd41d96.png">